resources/guides/uwinsite_guide.md

```diff
@@ -0,0 +1,236 @@
+5. From the Search Results, select a course.
+6. On the **Course Information** page you will see all class sections for the term you
+selected. Some classes have one section while others have multiple sections. Under
+**Class Selection**, you can view the time, date, instructor, location and available seats
+for each section of the class.
+![img](../../static/img/uwinsite_guide/course_info_and_class_selection.jpg)
+7. Select a class section. You can click anywhere on the class except for the Lecture or
+Lab link to select it.
+8. You’ll now be directed to the confirmation process:
+    1. Review **Class Selection** and click **Next**
+    2. Review **Class Preferences**
+        + This is where you have the option to add yourself to the waitlist if the class is
+        full. Select **Yes** if this will not put you in course overload (*normally 5 classes, but
+        check your program*). 
+        + Select **No** if having this course on the waitlist would put you in course overload.
+        You will instead need to use **Swap Classes** so UWinsite Student knows what
+        course to drop if you are to reach the top of the waitlist.
+    3. Click **Accept**
+    4. On the **Enroll** or **Add to Cart** page, click the **Add to Shopping Cart** button and
+    then click **Next**.
+    5. On the **Review and Submit** page, review your selections and then click **Submit**.
+
+#### Registering in your Classes
+
+Registration at UWindsor is by appointment time. Registration appointment times are
+assigned based on program and level of study. Your registration appointment time is
+when you can start to register. It is important to register early to have a better chance
+of getting into the classes that you want to take. Registration does, however, remain
+open until the tenth day of classes so you may make changes to your courses so long
+as there is space.
+
+Once registration appointment times are assigned for a term, you can find yours under
+**My Academics -> Enrolment Deadlines**. Your registration appointment time will also be
+sent to your UWindsor email account.
+
+**When your registration appointment time opens:**
+
+1. On the Student Homepage, click the **Manage Classes** tile.
+2. Click the **Shopping Cart** tab in the left navigation menu.
+3. Click the radio button next to each of your preferred classes and then click the **Enroll** button.
+    + If you are successfully enrolled in a class, you will see a **green check mark** beside it.
+    + If you are not enrolled in a class, you will see a **red X** beside it. 
+4. There are several reasons why you might not be successfully enrolled in a class:
+    + The class is full.
+    + Enrolment in the class is restricted for various reasons. For example: it is reserved
+    for students in particular majors or levels of study.
+    + You may be missing a prerequisite (this should not be the case IF you validated
+    your classes).
```

Comment:
```suggestion
4. There are several reasons why you might not be successfully enrolled in a class:
    + The class is full.
    + There is a time conflict.
    + Enrolment in the class is restricted for various reasons. For example: it is reserved
    for students in particular majors or levels of study.
    + You may be missing a prerequisite (this should not be the case IF you validated
    your classes).
```

Sometimes course times change to be conflicted or were always conflicting and the validator didnt catch it. I think its important to mention that case.